src/amo/components/Header/styles.scss

```diff
@@ -18,7 +18,7 @@
   }
 
   @include respond-to(extraLarge) {
-    grid-template-columns: max-content 1fr auto;
+    grid-template-columns: max-content 1fr 1fr;
```

Comment:
lol I was going to ask about this other `auto` in the other patch and thought "naww I'm sure he tested it and it's fine".

Reply:
Heh, yeah - saw some lovely visual regression tools demo'd at fronteers - stuff like this makes that look really useful.

Reply:
so we should forget about `auto` for any css grid stuff, right?